definitions/npm/express_v4.17.x/flow_v0.104.x-/express_v4.17.x.js

```diff
@@ -63,6 +63,7 @@ declare class express$Request extends http$IncomingMessage mixins express$Reques
   path: string;
   protocol: "https" | "http";
   query: { [name: string]: string | Array<string>, ... };
+  res?: express$Response | void;
```

Comment:
This should be fine, `?:` already represents optional (undefined), unless you did `void` because you meant to handle `null` in that case you'd want `?express$Response` or `express$Response | null`
```suggestion
  res?: express$Response;
```